src/Sarif.UnitTests/TestData/InsertOptionalDataVisitor/ExpectedOutputs/CoreTests_Hashes+TextFiles+ComprehensiveRegionProperties+RegionSnippets+ContextRegionSnippets+FlattenedMessages.sarif

```diff
@@ -41,8 +68,12 @@
           "uri": "file:///REPLACED_AT_TEST_RUNTIME/src/Sarif.UnitTests/TestData/"
         }
       },
-      "files": [
+      "artifacts": [
         {
+          "location": {
```

Comment:
>location [](start = 11, length = 8)

Here's the added data that prompted me to upgrade all these files to v2. We're getting close to doing this for everything. Looking forward to that.

Reply:
I don't actually understand this comment, but tell me some other time.

---
In reply to: [257484783](https://github.com/Microsoft/sarif-sdk/pull/1290#discussion_r257484783) [](ancestors = 257484783)